### PR TITLE
Use new LoadIconOverlayAsync in Bundles

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
Improved performance of Bundles item loading.

**Details of Changes**
Use `FileThumbnailHelper.LoadIconOverlayAsync()` instead of old slow mechanism.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: Files/UserControls/Widgets/LibraryCards.xaml.cs [hunk 1]
```diff
@@ -166,7 +164,7 @@ private async void DeleteLibrary_Click(object sender, RoutedEventArgs e)
 
         private async Task<byte[]> GetIcon(string path)
         {
-            return await LoadIconOverlay(path, 48u);
+            return (await FileThumbnailHelper.LoadIconOverlayAsync(path, 48u)).IconData;
```
Comment (gave92) [inline Files/UserControls/Widgets/LibraryCards.xaml.cs:167]: Since here you don't care about the overlay, if you wish there is also the `FileThumbnailHelper.LoadIconWithoutOverlayAsync` which only returns the icon

File: Files/ViewModels/Widgets/Bundles/BundleItemViewModel.cs [hunk 2]
```diff
@@ -152,12 +151,8 @@ public async void UpdateIcon()
                 {
                     if (Path.EndsWith(".lnk"))
                     {
-                        var (IconData, OverlayData, IsCustom) = LoadIconOverlay(Path, 24u);
-
-                        await CoreApplication.MainView.DispatcherQueue.EnqueueAsync(async () =>
-                        {
-                            Icon = await IconData.ToBitmapAsync();
-                        });
+                        var (IconData, OverlayData, IsCustom) = await FileThumbnailHelper.LoadIconOverlayAsync(Path, 24u);
```
Comment (gave92) [inline Files/ViewModels/Widgets/Bundles/BundleItemViewModel.cs:154]: Same here

File: Files/UserControls/Widgets/LibraryCards.xaml.cs (hunk 1 above)
Comment (d2dyno1) [reply Files/UserControls/Widgets/LibraryCards.xaml.cs:167]: Thanks

File: Files/ViewModels/Pages/WidgetsPageViewModel.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 namespace Files.ViewModels.Pages
 {
-    public class YourHomeViewModel : ObservableObject, IDisposable
+    public class WidgetsPageViewModel : ObservableObject, IDisposable
```
Comment (yaichenbaum) [inline Files/ViewModels/Pages/WidgetsPageViewModel.cs:13]: Can you revert this change? It doesn't appear to be related to the current pull request.

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

**Review by gave92 — APPROVED** [review]
